core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -34,63 +34,68 @@
  */
 public class RemovedKey {
 
-  private static final String V2_REMOVED = "v2.0 removed the ability to configure this parameter.";
+  private static final String V2_0_0_REMOVED = "this parameter is removed since v2.0.0.";
 
   private static final Map<String, String> REMOVED_KEYS = new HashMap<String, String>(20) {
     {
       put(Name.WEB_TEMP_PATH, "The alluxio web UI overhaul in v2.0 removed this parameter.");
-      put(Name.UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS, V2_REMOVED);
-      put(Name.SWIFT_USE_PUBLIC_URI_KEY, V2_REMOVED);
-      put(Name.MASTER_CLIENT_SOCKET_CLEANUP_INTERVAL, V2_REMOVED);
-      put(Name.MASTER_ACTIVE_UFS_SYNC_RETRY_TIMEOUT, V2_REMOVED);
-      put(Name.MASTER_ACTIVE_UFS_SYNC_BATCH_INTERVAL, V2_REMOVED);
+      put(Name.UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS, V2_0_0_REMOVED);
+      put(Name.SWIFT_USE_PUBLIC_URI_KEY, V2_0_0_REMOVED);
+      put(Name.MASTER_CLIENT_SOCKET_CLEANUP_INTERVAL, V2_0_0_REMOVED);
+      put(Name.MASTER_CONNECTION_TIMEOUT, V2_0_0_REMOVED);
+      put(Name.MASTER_HEARTBEAT_INTERVAL, V2_0_0_REMOVED + " Use "
+          + PropertyKey.Name.MASTER_WORKER_HEARTBEAT_INTERVAL + " instead.");
       put(Name.MASTER_JOURNAL_FORMATTER_CLASS, "v2.0 removed the ability to specify the master "
           + "journal formatter");
-      put(Name.LOGSERVER_LOGS_DIR, V2_REMOVED);
```

Comment:
these two LOGSERVER related are actually still used. @calvinjia

Reply:
plus, the logserver configuration is not fully working. do you want to fix it in 2.0?